cmd/main.go

```diff
@@ -1,32 +1,48 @@
 package main
 
 import (
-	"fmt"
+	"context"
+	"log"
 	"time"
 
 	"gopkg.in/tucnak/telebot.v2"
 
-	"telegram-budget-bot/internal/bot"
-	"telegram-budget-bot/internal/config"
-	"telegram-budget-bot/internal/logger"
-	"telegram-budget-bot/internal/model"
-	"telegram-budget-bot/internal/storage"
+	"github.com/jackc/pgx/v4/pgxpool"
+
+	"budget-bot/internal/bot"
+	"budget-bot/internal/config"
+	"budget-bot/internal/logger"
+	"budget-bot/internal/model"
+	"budget-bot/internal/storage"
 )
 
 var userSessions = make(map[int64]*model.UserSession)
 
 func main() {
-	log := logger.New()
 	cfg, err := config.LoadConfig()
 	if err != nil {
-		log.Fatalf("Ошибка при загрузке конфигурации: %v", err)
+		log.Fatalf("error loading configuration: %v", err)
```

Comment:
log messages should have the same structure. If you decided to start each message with `error ...` then apply this way to all other log messages over the project